src/parse/features/suggestions.rs

```diff
@@ -34,7 +34,7 @@ where
 }
 
 /// Returns a suffix that can be empty, or is the standard 'did you mean' phrase
-pub fn did_you_mean_flag_suffix<I, T>(
+pub(crate) fn did_you_mean_flag_suffix<I, T>(
     arg: &str,
```

Comment:
This file is the one I'm not sure about. Do we want to export these functions? They've got doc comments.

Reply:
No need.